inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -130,11 +132,61 @@ void visit_exec_graph_node(pugi::xml_node& data, std::string& node_type_name,
     }
 }
 
+NodeVector get_ordered_ops(const ngraph::Function& f) {
+    NodeVector op_order;
+    std::stack<std::shared_ptr<Node>> stack;
+    std::unordered_set<Node*> visited;
+
+    auto has_unvisited_inputs = [](const std::unordered_set<Node*>& visited,
+                                   const std::vector<Input<Node>>& inputs) {
+        for (auto input = inputs.crbegin(); input != inputs.crend(); ++input) {
+            const auto source_output =
+                input->get_source_output().get_node_shared_ptr();
+            if (visited.count(source_output.get()) == 0) {
+                return true;
+            }
+        }
+        return false;
+    };
+
+    const auto& results = f.get_results();
+    for (auto result = results.crbegin(); result != results.crend(); ++result) {
+        stack.push(*result);
+    }
+
+    const auto& sinks = f.get_sinks();
+    for (auto sink = sinks.crbegin(); sink != sinks.crend(); ++sink) {
+        stack.push(*sink);
+    }
+
+    while (stack.size()) {
```

Comment:
```suggestion
    while (!stack.empty()) {
```

Reply:
Applied in [this](https://github.com/openvinotoolkit/openvino/pull/3156/commits/1b80d2856153c7ccad62f8cd9146934a34529b23) commit.